components/lwip/Kconfig

```diff
@@ -537,6 +537,24 @@ menu "LWIP"
         default 0x0 if LWIP_TCPIP_TASK_AFFINITY_CPU0
         default 0x1 if LWIP_TCPIP_TASK_AFFINITY_CPU1
 
+    menu "SNMP"
+        menuconfig LWIP_SNMP_SUPPORT
+            bool "Enable SNMP support"
+            default n
```

Comment:
When `LWIP_SNMP_SUPPORT` enabled, getting error messages:
```
#error LWIP_SNMP MIB2 needs LWIP_STATS (for MIB2)
```

Possibly we can use `select` to enable the other option in Kconfig.

Could we use `LWIP_SNMP_SUPPORT` without `LWIP_STATS` ?